core/math/vector2.h

```diff
@@ -181,6 +159,31 @@ struct _NO_DISCARD_ Vector2 {
 	}
 };
 
+_FORCE_INLINE_ real_t &Vector2::operator[](int p_idx) {
+	DEV_ASSERT((unsigned int)p_idx < 2);
+	return coord[p_idx];
+}
+_FORCE_INLINE_ const real_t &Vector2::operator[](int p_idx) const {
```

Comment:
```suggestion

_FORCE_INLINE_ const real_t &Vector2::operator[](int p_idx) const {
```
If we keep them here, leave a line between each implementation